mtenn/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.2.0"
+__version__ = "0.6.0+0.gc18b034.dirty"
```

Comment:
should this just be `0.6.0`?

Reply:
I have fixed this to no longer even need to write this file.